Upstatement/reviewerator
# Feedback responses

# What's in this PR
* change back to light color palette
* increase max width of chart
* add ability to set a skill as a target by clicking on the current level block (click again to remove)
* add Dia font files
<img width="1705" alt="Screen Shot 2019-12-20 at 1 35 46 PM" src="https://user-images.githubusercontent.com/18580335/71283315-ce4ee980-232d-11ea-8880-08ea7efb8e58.png">
<img width="1705" alt="Screen Shot 2019-12-20 at 1 35 42 PM" src="https://user-images.githubusercontent.com/18580335/71283316-cee78000-232d-11ea-9964-d602326ce3f3.png">

# To Test
click on a skill block to set the level, then click again to set it as a target. click once more to remove the target.

on a machine without Dia installed, make sure that the fonts display correctly.
